src/Platforms/PostgreSQL94Platform.php

```diff
@@ -380,7 +380,7 @@ private function getTableWhereClause($table, $classAlias = 'c', $namespaceAlias
             [$schema, $table] = explode('.', $table);
             $schema           = $this->quoteStringLiteral($schema);
         } else {
-            $schema = 'ANY(current_schemas(false))';
+            $schema = "CURRENT_SCHEMA";
```

Comment:
This will work only if the table is found in the first schema in the search path. What if it's not?

Reply:
I added tests to show that it still works with the table in the different schema.

Reply:
The `testGetDetailsStillWorkingWithOneTable` test only proves that the query doesn't fail but doesn't prove that it works. Adding a few assertions will prove that it doesn't work:
1. The introspected table name is `migrations` but it should be `001_test.migrations`.
2. It has no columns.

Reply:
Added test for `001_test.migrations`. 

Of course it will not work if that table does not exist eg. if I change `migrations` to `"001_test".migrations` below:
```php
    public function testGetDetailsStillWorkingWithOneTable3(): void
    {
        $createTableSQL = 'CREATE TABLE IF NOT EXISTS migrations(a text);';
        $this->connection->executeStatement($createTableSQL);
        $createTableSQL = 'DROP TABLE IF EXISTS "001_test".migrations;';
        $this->connection->executeStatement($createTableSQL);
        $this->connection->executeStatement('SET search_path TO "001_test",public');

        $databaseTable = $this->schemaManager->listTableDetails('"001_test".migrations');

        self::assertNotNull($databaseTable);
    }
```

When the `migrations` table does not have columns it works for me for me. I added a test for it too. I thought it did not work, but my syntax was wrong: `CREATE TABLE IF NOT EXISTS migrations;`, since there's no `()` after `migrations`.